package.json

```diff
@@ -24,7 +24,8 @@
     "deploy": "gh-pages -d build",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prettier": "prettier --write ."
```

Comment:
This is the main change in this PR, it adds `yarn prettier` which will format the code. The rest of the changes here are from running this command to format the code the first time.

VSCode should be configured to run Prettier for you automatically on save but if it's not then check out the instructions [here](https://prettier.io/docs/en/editors.html#visual-studio-code)!